uisdk/src/main/java/com/karhoo/uisdk/util/CurrencyExtensions.kt

```diff
@@ -12,7 +12,14 @@ fun Currency.intToRangedPrice(lowPrice: Int, highPrice: Int, locale: Locale = Lo
 }
 
 fun Currency.intToPriceNoSymbol(price: Int, locale: Locale = Locale.getDefault()): String {
-    return this.formatted(price, locale, includeCurrencySymbol = false)
+    val cost = Integer.toString(price)
+    val low = defaultFractionDigits
+
+    val costHiEndIndex = if (cost.length - low > 0) cost.length - low else 0
+    val costHi = if (costHiEndIndex == 0) "0" else cost.substring(0, costHiEndIndex)
+    val costLow = cost.substring(costHiEndIndex)
+
+    return String.format("%s.%s", costHi, costLow)
```

Comment:
The naming of the variables lead to the idea it's a price range
Also, maybe we could modify the Currency.formatted function to output this format also, instead of having two different methods

Reply:
Unfortunately, the code is not working for Braintree using the French locale. 
We need to look into it later, but I would like to revert back to the working code. 
Next time we should verify the changes on different locales and not only rely on the Unit Tests